docs/configuration/network_filters/redis_proxy_filter.rst

```diff
@@ -0,0 +1,91 @@
+.. _config_network_filters_redis_proxy:
+
+Redis proxy
+===========
+
+Redis :ref:`architecture overview <arch_overview_redis>`.
+
+.. code-block:: json
+
+  {
+    "name": "redis_proxy",
+    "type": "read",
+    "config": {
+      "cluster_name": "...",
+      "conn_pool": "{...}",
+      "stat_prefix": "..."
+    }
+  }
+
+cluster_name
+  *(required, string)* Name of cluster from cluster manager.
+  See the :ref:`configuration section <arch_overview_redis_configuration>` of the architecture overview for recommendations
+  on configuring the backing cluster.
```

Comment:
Per offline convo we should probably do a PR to just block cluster config that does not configure ring hash, or at least warn.